src/ui/trigger_pattern_edit.ui

```diff
@@ -177,14 +177,7 @@
     </widget>
    </item>
    <item>
-    <widget class="QLineEdit" name="lineEdit_pattern">
-     <property name="maximumSize">
-      <size>
-       <width>16777215</width>
-       <height>33</height>
-      </size>
-     </property>
-    </widget>
+    <widget class="SingleLineTextEdit" name="lineEdit_pattern"/>
```

Comment:
Also, given that you are **changing** the underlying type you should change the name/objectName for this from 
```suggestion
    <widget class="SingleLineTextEdit" name="singleLineTextEdit_pattern"/>
```
though this will ripple through and cause more changes it will avoid the sort of confusion that the `(QListWidget) dlgConnectionProfiles::profiles_tree_widget` causes (is it a "`QTreeWidget`" nope, it is a "`QListWidget`"... :confused:)...

Reply:
1. ah, thanks about the designer... didn't catch it pulled more info out, please check this fixes your Chrome issue @vadi2

2. I knew I should have renamed this widget from the get go :)

both done